docs/storage-management.asciidoc

```diff
@@ -3,67 +3,159 @@
+* Indexing 10 transactions per second for 1 hour, each transaction with 10 spans, each span with 10 frames, results in 396,000 documents. These documents use around **200 Mb** of disk space. 
+* Indexing 25 transactions per second for 1 hour, each transaction with 25 spans, each span with 25 frames, results in 2,340,000 documents. These documents use around **1.2 Gb** of disk space.
+
+NOTE: These examples were indexing the same data over and over with minimal variation. Because of that, the compression ratios of 80-90% are somewhat optimistic.
+
+[[processing-performance]]
+== Processing and performance
+
+APM Server performance depends on a number of factors: memory and CPU available,
+network latency, transaction sizes, workload patterns,
+agent and server settings, versions, and protocol.
+
+Let's look at a simple example that makes the following assumptions:
+
+* The load is generated in the same region as where APM Server and Elasticsearch are deployed.
+* We're using the default settings in cloud.
+* A small number of agents are reporting.
+
+This leaves us with relevant variables like payload and instance sizes.
+See the table below for approximations.
+As a reminder, events are
+{apm-overview-ref-v}/transactions.html[transactions] and
+{apm-overview-ref-v}/transaction-spans.html[spans].
+
+[options="header"]
+|=======================================================================
+|Transaction/Instance |512Mb Instance |2Gb Instance |8Gb Instance
+|Small transactions
+
+_5 spans with 5 frames each_ |600 events/second |1200 events/second |4800 events/second 
+|Medium transactions
+
+_15 spans with 15 frames each_ |300 events/second |600 events/second |2400 events/second
+|Large transactions
+
+_30 spans with 30 frames each_ |150 events/second |300 events/second |1400 events/second
+|=======================================================================
+
+In other words, a 512 Mb instance can process \~3 Mbs per second,
+while an 8 Gb instance can process ~20 Mbs per second.
+
+APM Server is CPU bound, so it scales better from 2 Gb to 8 Gb than it does from 512 Mb to 2 Gb. This is because larger instance types come with much more computing power.
+
+Don't forget that the APM Server is stateless. Several instances running do not need to know about each other. This means that with a properly sized Elasticsearch instance, APM Server scales out linearly.
+
+NOTE: RUM deserves special consideration. The RUM agent runs in browsers, and there can be many thousands reporting to an APM Server with very variable network latency. 
 
 [[reduce-storage]]
 == Reduce storage
+
 The amount of storage for APM data depends on several factors:
-how many services you are instrumenting, how much traffic the services see, agent and server settings,
-and for how long you keep monitoring data.
+the number of services you are instrumenting, how much traffic the services see, agent and server settings,
+and the length of time you keep monitoring data.
```

Comment:
actually, as non native speaker this is confusing at first sigh since "monitoring" can be a verb or an adjective... consider "for how long you keep data"